migration/20170224-2-copy-collection-configuration-into-database.py

```diff
@@ -113,11 +113,11 @@ def convert_one_click(_db, library):
         name="OneClick"
     )
     library.collections.append(collection)
-    collection.password = basic_token
+    collection.external_integration.password = basic_token
     collection.external_account_id = library_id
-    collection.url = url
-    collection.set_setting("ebook_loan_length", ebook_loan_length)
-    collection.set_setting("eaudio_loan_length", eaudio_loan_length)
+    collection.external_integration.url = url
+    collection.external_integration.set_setting("ebook_loan_length", ebook_loan_length)
+    collection.external_integration.set_setting("eaudio_loan_length", eaudio_loan_length)
```

Comment:
They're still being accessed in this version of the codebase, but they've been removed in multi-tenancy. I can create a migration to remove them from the multi-tenancy feature branch, buuuuut I'm going to leave them here.